pkg/reconciler/taskrun/taskrun.go

```diff
@@ -196,7 +196,7 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, tr *v1.TaskRun) pkgrecon
 		logger.Errorf("Reconcile: %v", err.Error())
 		if errors.Is(err, sidecarlogresults.ErrSizeExceeded) {
 			cfg := config.FromContextOrDefaults(ctx)
-			message := fmt.Sprintf("TaskRun \"%q\" failed: results exceeded size limit %d bytes", tr.Name, cfg.FeatureFlags.MaxResultSize)
+			message := fmt.Sprintf("[User error] TaskRun \"%q\" failed: results exceeded size limit %d bytes", tr.Name, cfg.FeatureFlags.MaxResultSize)
```

Comment:
can we use the `userErrorLabel` const here (and same below)?

Reply:
Updated. But for the rest, I think they shall be fine to not import the errors pkg in test? WDYT

Reply:
why importing the error pkg to test is an concern here?

Reply:
Updated to be using imported consts. PTAL